bin/lib/create.js

```diff
@@ -98,6 +106,16 @@ module.exports.run = function (argv) {
     return Q.resolve();
 };
 
+function recursiveCreateDirectory(targetPath) {
+    var parent = path.join(targetPath, '..');
+    if (!fs.existsSync(parent)) {
+        recursiveCreateDirectory(parent);
+    }
+
+    console.log('Creating directory "' + targetPath + '"');
```

Comment:
Heh... Do we really want to be notified about each directory we're creating in the path? If we need any notification, I'd prefer just a single one (so at the call point rather than here).